dbt/include/athena/macros/materializations/models/table/create_table_as.sql

```diff
@@ -14,6 +14,7 @@
 
   {%- set location_property = 'external_location' -%}
   {%- set partition_property = 'partitioned_by' -%}
+  {%- set work_group_output_location = adapter.get_work_group_output_location() -%}
```

Comment:
No need for seed also ? especially in `athena__create_csv_table` ?

Reply:
Seeds work for me, I only get:
```
StartQueryExecution operation: The Create Table As Select query failed because it was submitted with an 'external_location' property to an Athena Workgroup that enforces a centralized output location for all queries. Please remove the 'external_location' property and resubmit the query.
```

when running models. Maybe because seeds create an external table?

Reply:
Indeed seeds don't need external location, as we just create metadata, so no CTAs involved.